content/docs/ui/account-and-settings/upgrading-your-account.md

```diff
@@ -64,3 +62,21 @@ Send to up to 2,000 contacts and 6,000 Emails per month and test all the feature
 Take control of your sending reputation and deliverability with a dedicated IP address, provided with every Pro, Premier and Advanced plan. [Learn More](https://sendgrid.com/blog/shared-and-dedicated-ips-which-should-you-choose/).
 
 </call-out>
+
+### Choose a plan
+
+*Email API*
+
+1. Decide the plan you upgrade to by comparing your current contact storage usage and email sends to the available plans [here](https://sendgrid.com/pricing).
```

Comment:
Remove Contact storage reference: 

Decide the plan you upgrade to by reviewing features and comparing your current email send volume to the available plans [here](https://sendgrid.com/pricing).

Reply:
I think we need a callout somewhere in this section reminding them they will not have access to upgrade until they complete the steps outlined above.